cmd/apichecker/apicheck.go

```diff
@@ -0,0 +1,1225 @@
+							}
+						}
+					}
+				}
+			}
+		default:
+			continue
+		}
+	}
+	return resp, nil
+}
+
+// HTMLScrapeYobit gets the check string for Yobit Exchange
+func HTMLScrapeYobit(htmlData *HTMLScrapingData) ([]string, error) {
+	var resp []string
+	temp, err := http.Get(htmlData.Path)
+	if err != nil {
+		return resp, err
+	}
+	tokenizer := html.NewTokenizer(temp.Body)
+	var case1, case2, case3 bool
+loop:
+	for {
+		next := tokenizer.Next()
+		switch next {
+		case html.ErrorToken:
+			break loop
+		case html.StartTagToken:
+			token := tokenizer.Token()
+			if token.Data == htmlData.TokenData {
+				for _, x := range token.Attr {
+					if x.Key == htmlData.Key {
+						switch x.Val {
+						case "n4":
+							inner := tokenizer.Next()
+							if inner == html.TextToken {
+								if string(tokenizer.Text()) == "v2" {
+									case1 = true
+								}
+							}
+						case "n5":
+							inner := tokenizer.Next()
+							if inner == html.TextToken {
+								tempStr := string(tokenizer.Text())
+								if tempStr == "v3" {
+									case2 = true
+									resp = append(resp, tempStr)
+								}
+							}
+						case "n6":
+							inner := tokenizer.Next()
+							if inner == html.TextToken {
+								tempStr := string(tokenizer.Text())
+								if tempStr != "v4" {
+									case3 = true
+								}
+								if case1 && case2 && case3 {
+									return resp, nil
+								}
+								resp = append(resp, tempStr)
+							}
+						}
+					}
+				}
+			}
+		default:
+			continue
+		}
+	}
+	return resp, nil
+}
+
+// HTMLScrapeLocalBitcoins gets the check string for Yobit Exchange
+func HTMLScrapeLocalBitcoins(htmlData *HTMLScrapingData) (string, error) {
+	temp, err := http.Get(htmlData.Path)
+	if err != nil {
+		return "", err
+	}
+	a, err := ioutil.ReadAll(temp.Body)
+	if err != nil {
+		return "", err
+	}
+	abody := string(a)
+	r, err := regexp.Compile(htmlData.RegExp)
+	if err != nil {
+		return "", err
+	}
+	str := r.FindString(abody)
+	sha := crypto.GetSHA256([]byte(str))
+	return crypto.HexEncodeToString(sha), nil
+}
+
+// GetListsData gets required data for all the lists on the given board
+func GetListsData(idBoard string) ([]ListData, error) {
+	var resp []ListData
+	err := SendHTTPRequest(pathGetAllLists+idBoard+apiKey+apiToken, &resp)
+	if err != nil {
+		return resp, err
+	}
+	return resp, nil
+}
+
+// CreateNewCard creates a new card on the list specified
+func CreateNewCard(fillData CardFill) error {
+	params := url.Values{}
+	params.Set("idList", fillData.ListID)
+	if fillData.Name != "" {
+		params.Set("name", fillData.Name)
+	}
+	if fillData.Desc != "" {
+		params.Set("desc", fillData.Desc)
+	}
+	if fillData.Pos != "" {
+		params.Set("pos", fillData.Pos)
+	}
+	if fillData.Due != "" {
+		params.Set("due", fillData.Due)
+	}
+	if fillData.MembersID != "" {
+		params.Set("idMembers", fillData.MembersID)
+	}
+	if fillData.LabelsID != "" {
+		params.Set("idLabels", fillData.LabelsID)
+	}
+	_, err := common.SendHTTPRequest(http.MethodPost,
```

Comment:
make sure you test a bad case on this, to see what comes back. I just want to make sure there might not be an error in the body. There shouldn't be when you go off the API docs.

Reply:
Also if you can use a function that takes in a verbose variable might be better in debugging all these calls in the future.